hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -214,6 +214,17 @@ systemctl enable --now load-br-netfilter
 
 systemctl restart NetworkManager
 
+echo 'GRUB_TIMEOUT=1
+GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/system-release)"
+GRUB_DEFAULT=saved
+GRUB_DISABLE_SUBMENU=true
+GRUB_TERMINAL_OUTPUT="console"
+GRUB_CMDLINE_LINUX="console=ttyS0,115200n8 no_timer_check net.ifnames=0 hugepagesz=2M hugepages=16 crashkernel=1G-4G:192M,4G-64G:256M,64G-:512M"
```

Comment:
AFAIU `net.ifnames=0` means to not let kernel rename nedev names i.e use ethX style netdev names.
do we need this ?

Reply:
I just copy the default one from inside the machine and add the hugepages part only :)